config/pipelines/high_throughput_cardinal.yml

```diff
@@ -1,13 +1,16 @@
 ---
 # Initial transfer from tube to plate, as entry for one or other of below pipelines. Happens without submissions / requests.
 Cardinal Entry A:
+  pipeline_group: Cardinal A
```

Comment:
Change `Cardinal A` to `Cardinal`, or `Cardinal library prep`

Reply:
Gone with `Cardinal library prep`